drivers/bbdmx.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const dgram = require('dgram');
+
+function BBDMX(device_id, options) {
+	const self = this;
+	self.options = options || {};
+	self.universe = new Buffer(512);
+	self.universe.fill(0);
+	self.host = device_id || '127.0.0.1';
+	if (typeof options === 'undefined') {
+		self.port = 9930;
+	} else {
+		self.port = options.port || 9930;
+	}
+	self.dev = dgram.createSocket('udp4');
+	self.sleepTime = 24;
+	self.start();
+}
+
+BBDMX.prototype.send_universe = function() {
+	let messageBuffer = new Buffer(`${this.universe.length} `);
```

Comment:
could you adjust this to `new Buffer(this.universe.length + ' ')` so it'll still work with oder node versions?